content/docs/1.7.3/important-notes/_index.md

```diff
@@ -40,6 +41,45 @@ Please see [here](https://github.com/longhorn/longhorn/releases/tag/v{{< current
   - [Snapshot Creation Time As Shown in the UI Occasionally Changes](#snapshot-creation-time-as-shown-in-the-ui-occasionally-changes)
   - [Unable To Revert a Volume to a Snapshot Created Before Longhorn v1.7.0](#unable-to-revert-a-volume-to-a-snapshot-created-before-longhorn-v170)
 
+## Warning
+
+The longhorn-share-manager v1.7.3 is impacted by a [regression issue](https://github.com/nfs-ganesha/nfs-ganesha/issues/1132) in nfs-ganesha v6.0+, which causes unexpected "permission denied" errors when non-root users access RWX volumes. To resolve this issue, replace `longhorn-share-manager:v1.7.3` with the hotfixed image `longhorn-share-manager:v1.7.3-hotfix-1`.
```

Comment:
```suggestion
The longhorn-share-manager v1.7.3 is impacted by a [regression issue](https://github.com/nfs-ganesha/nfs-ganesha/issues/1132) in nfs-ganesha v6.0+, which causes unexpected "permission denied" errors when non-root users access RWX volumes. To resolve this issue, replace `longhorn-share-manager:v1.7.3` with the hotfix image `longhorn-share-manager:v1.7.3-hotfix-1`.
```

Reply:
I think hotfixed image is a correct one.

https://www.google.com/search?q=%22hotfixed+image%22&oq=%22hotfixed+image%22&gs_lcrp=EgZjaHJvbWUyBggAEEUYOTIGCAEQLhhA0gEHNjIzajBqMagCALACAA&sourceid=chrome&ie=UTF-8